synology-packages.md

```diff
@@ -13,33 +13,32 @@ dateCreated: 2022-05-06T13:45:19.731Z
 - The Servarr Team now creates and maintains Synology Packages
 - Installation instructions are noted below for the specific DSM versions
 
-> Generally the existing SynoCommunity versions are likely not compatible with the Servarr versions without some hoop jumping. This message to be updated and clarfied in the future.
+> Generally the existing SynoCommunity versions are likely not compatible with the Servarr versions without some hoop jumping. This means it would be wise te delete the old package after doing a [backup of your database](https://wiki.servarr.com/en/radarr/faq#how-do-i-backuprestore-radarr). Which can be done through the web interface of the starr app.
```

Comment:
Won't this cause issues since the radarr port was different?
@ta264  what did we decide here?

Reply:
Didn't think about this.
Need to ensure the ports match then before installation?
Are you guys going to use the default radarr port?

Reply:
Add a blurb about the port change have the other fix ups and we're good.